mne/preprocessing/tests/test_ssp.py

```diff
@@ -122,4 +125,31 @@ def test_compute_proj_parallel():
                                    bads=['MEG 2443'])
     assert_array_almost_equal(projs, projs_2, 10)
 
+
+def test_compute_proj_ctf():
+    """Trivial Test to show that projector code completes on CTF data"""
+    raw = read_raw_ctf(ctf_fname)
+    raw.load_data()
+
+    for average in [False, True]:
+        n_projs_init = len(raw.info['projs'])
+        projs, events = compute_proj_eog(raw, n_mag=2, n_grad=2, n_eeg=2,
+                                         average=average,
+                                         ch_name='EEG059',
+                                         avg_ref=True, no_proj=False,
+                                         l_freq=None, h_freq=None,
+                                         reject=None, tmax=dur_use,
+                                         filter_length=6000)
+        assert_true(len(projs) == (5 + n_projs_init))
```

Comment:
With pytest it's better to do now `assert len(projs) == ...` directly, it gives nicer error messages than `assert_true`